pallets/gear/src/lib.rs

```diff
@@ -96,6 +96,8 @@ pub mod pallet {
         InitFailure(MessageInfo, Reason),
         /// Program initialized.
         InitSuccess(MessageInfo),
+        /// Program initialization deferred.
+        InitWait(MessageInfo),
```

Comment:
No need for this event

Reply:
Emit `InitSUccess` instead?

Reply:
Yes, but once init is success, not on wait :)